charts/kubescape-operator/templates/node-agent/daemonset.yaml

```diff
@@ -112,10 +114,17 @@ spec:
             {{- end }}
           securityContext:
             runAsUser: 0
-            privileged: true
```

Comment:
`privileged: {{ .Values.nodeAgent.privileged }}`

Reply:
This will need to be added to the values file, by default it should be `false`

Reply:
Done